Sources/PagecallSDK/NativeBridge.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  NativeBridge.swift
+//
+//
+//  Created by 록셉 on 2022/07/26.
+//
+
+import Foundation
+import WebKit
+
+class NativeBridge {
+    let webview: WKWebView
+
+    init(webview: WKWebView) { self.webview = webview }
+
+    func emit(eventName: String, data: Data) {
+        let string = String(data: data, encoding: .utf8)
+        if let string = string {
```

Comment:
[Optional]
```suggestion
        if let string = String(data: data, encoding: .utf8) {
```

Reply:
resolved in d418d82534e250181cbcd21410bf0139a2888463